### ISSUE TITLE
Line breaks are not displayed

### ISSUE THREAD

**Issue by javiavid** [ckRobinson/multi-column_markdown#1]
If the text has line breaks they are not displayed in reading mode
Is it possible to improve this?

**Comment by javiavid** [comment]
Thanks for your response, I send you an example

![image](https://user-images.githubusercontent.com/7627504/150966899-20a06997-5505-4feb-a106-5ee9dc85b6a9.png)


[test multi column plugin.md](https://github.com/ckRobinson/multi-column_markdown/files/7933062/test.multi.column.plugin.md)

**Comment by ckRobinson** [comment]
Strange, that doesn't happen on the two machines I've tested on. And your test file renders normally for me as well:

<img width="1687" alt="Multi-Column_Markdown_Test_File" src="https://user-images.githubusercontent.com/70980846/151019881-8d060b11-760b-428c-8dc0-6579af9d3868.png">

Are there other plugins you have installed that could possibly be causing a conflict?

**Comment by javiavid** [comment]
I think the problem is because of CSS. In some themes it works well and in others it doesn't.
For example in minimal, palatine and spectrum it looks correct, in the rest that I have tried, including the one that comes by default it looks bad. I've tried 25 themes and it only looks good on 3 or 4.


![image](https://user-images.githubusercontent.com/7627504/151132199-51e1a22a-1da4-494e-84bb-f61cbc7dbc6c.png)
